sources/web/datalab/static/datalab.css

```diff
@@ -575,6 +575,9 @@ div.selected .CodeMirror-linenumber, div#texteditor-container .CodeMirror-linenu
 div.modal-dialog pre, div.modal-dialog code, #help code, #help pre {
   font-size: 13px !important;
 }
+#help {
+  font-family: 'Source Code Pro';
```

Comment:
Can you add monospace after the specific font, in case that fails to download or not available for whatever reason?

Reply:
Done.